pkg/repoowners/repoowners.go

```diff
@@ -133,6 +134,19 @@ func NewClient(
 // RepoAliases defines groups of people to be used in OWNERS files
 type RepoAliases map[string]sets.String
 
+// ForeignAlias contains a
+type ForeignAlias struct {
+	Org  string `json:"org"`
+	Repo string `json:"name"`
+	Ref  string `json:"ref,omitempty"`
```

Comment:
Should this also be an option? I thought we always want to take the value from master/main branch, right? Curious ...

Reply:
Well, I need to provide a reference to GetFile anyway so I thought I could as well make it configurable.

Reply:
So trying to understand, if I open a PR in the repo and edit the alias, and it is still not merged into main/master, is it possible for me to have approving rights?

Reply:
No